### DISCUSSION TITLE
Click to View Recommended Videos (Up Next)?

### DISCUSSION THREAD

**Discussion by lothar-cell** [FreeTubeApp/FreeTube#4850] [Q&A]
Is there a such an option in settings, like "Click to View Comments"? 
Distraction Free Settings hide Recommended Videos completely, leave on option to unhide with a click.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No this doesnt exists and wouldnt really make sense to implement.

**Reply by lothar-cell** [reply]
Nice chatting with you as always:D
Anyway, then I guess it doesn't make sense in Piped either?
<img width="1440" alt="Screenshot 2024-04-04 at 06 49 07" src="https://github.com/FreeTubeApp/FreeTube/assets/2356354/dbafd7f0-4ab2-4917-9456-f5e367d1a628">
